src/documents/TextToSpeechAPI.md

```diff
@@ -372,11 +376,11 @@ The events fired by the Text To Speech component are described below.
     </tbody>
 </table>
 
-## utteranceOpts` Option ##
+## <a name="utteranceopts-option"></a>`model.utteranceOpts` Configuration ##
 
 <div class="infusion-docs-note"><strong>Note:</strong> If needed, please read the [Component Configuration Options](ComponentConfigurationOptions.md) document for a full description of infusion component options.</div>
```

Comment:
Could you update the syntax used for the link here. It's in an HTML block so needs to use `<a>`?